### PR TITLE
 Change type of IDs from int64 to uuid 

### PR DESCRIPTION
**Pull request by ghost**
Stored as native uuids in postgres, and binary blobs in sqlite. Should be more or less indistinguishable in terms of performance and functionality.

### CODE REVIEW

File: pkg/models/model_performer.go [hunk 1]
```diff
@@ -298,15 +308,6 @@ func (p *Performer) CopyFromCreateInput(input PerformerCreateInput) error {
...
 
-	if input.Image != nil {
-		var err error
-		p.Image, err = p.TranslateImageData(input.Image)
-
-		if err != nil {
-			return err
-		}
-	}
-
```
Comment (WithoutPants) [inline pkg/models/model_performer.go]: This looks unrelated to the uuid change. Why was this removed from here?

File: pkg/models/querybuilder_sql.go [hunk 2]
```diff
@@ -37,22 +38,18 @@ func handleStringCriterion(column string, value *StringCriterionInput, query *da
...
 
-	if err != nil {
-		return 0, err
-	}
-
-	return result.LastInsertId()
+	return err
 }
 
 func insertJoins(tx *sqlx.Tx, table string, joins interface{}) error {
```
Comment (WithoutPants) [inline pkg/models/querybuilder_sql.go]: Since `insertObject` no longer returns the id, I think we can rename this to `insertObjects`.

**Review by WithoutPants — CHANGES_REQUESTED** [review]
One minor change and a question.

File: pkg/models/model_performer.go (hunk 1 above)
Comment (ghost) [reply pkg/models/model_performer.go]: My bad, fixed.

File: pkg/models/querybuilder_sql.go (hunk 2 above)
Comment (ghost) [reply pkg/models/querybuilder_sql.go]: It still only inserts a single object so I'm not quite sure why we would change it?

Comment (WithoutPants) [reply pkg/models/querybuilder_sql.go]: This refers to line 55 - to rename `insertJoins` to `insertObjects`.

Comment (ghost) [reply pkg/models/querybuilder_sql.go]: Oh, gotcha, the diff confused me. I've updated the name.

**Review by WithoutPants — APPROVED** [review]
